app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  devise :database_authenticatable, :registerable, :validatable
```

Comment:
- [ ] Kindly add `confirmable` attributes on these devise attributes to help you set up email confirmations. __Before users can access the app, an email confirmation should be done__.
- [ ] I have also observed a repetition of these attributes on line 6, kindly fix this as well.

Reply:
Done, as advised